yb-voyager/cmd/analyzeSchema.go

```diff
@@ -259,6 +260,29 @@ func reportSummary() {
 	}
 }
 
+func addSummaryDetailsForIndexes() {
+	var indexesInfo []utils.IndexInfo
+	found, err := metaDB.GetJsonObject(nil, metadb.INDEXES_INFO_KEY, &indexesInfo)
+	if err != nil {
+		utils.ErrExit("analyze schema report summary: load indexes info: %s", err)
+	}
+	if found {
+		indexesPresent := lo.Keys(summaryMap["INDEX"].objSet)
+		unexportedIdxsMsg := "Indexes which are neither exported by yb-voyager as they are unsupported in YB and needs to be handled manually:\n"
+		unexportedIdxsPresent := 0
```

Comment:
bool instead of int?

Reply:
also, should we increment the invalidCount?

Reply:
I was having different opinion on that.
Since we are not exporting those indexes, so the object names reported also doesn't have these unexported indexes.
Till now, we have been considering invalidCount for objects which are exported and have issues but can be fixed.
This is a different case, so i thought just report in the details.
Let me know what do you think

Reply:
+1 to Sanyam, `invalidCount` for any object type is currently only for CREATE DDLs in the exported schema which have issues and are reported in the `Issues` section. I don't think it will make sense to add these unexported Indexes in the `invalidCount` of indexes as it won't be uniform with other type's `invalidCount` as well.